content/docs/12-enterprise-version/00-on-prem-system-requirements.md

```diff
@@ -66,19 +100,57 @@ The following points needs to be taken care while creating the Tags:
 
 </InfoBox>
 
-### Permissions
 
-The following permissions are required for the account used to install the platform:
+## Privilages 
+
+**Last Update**: August 18, 2022
```

Comment:
```suggestion
```
We should remove this. We don't specify anywhere else in the docs when a document was updated.